### PR TITLE
snap: migrate to core18

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `./runtests.sh static`?
- [ ] Have you successfully run `./runtests.sh tests/unit`?

-----

### CODE REVIEW

File: spread.yaml [hunk 1]
```diff
@@ -167,6 +172,56 @@ debug-each: |
...
+       libxslt-dev \
+       libyaml-dev \
+       python3-venv \
+       python3-yaml \
+       python3.6-dev \
+       rpm2cpio \
+       squashfs-tools \
+       subversion \
+       xdelta3
+     apt-mark auto \
```
Comment (cjp256) [inline spread.yaml]: indent

File: snapcraft/extractors/appstream.py [hunk 2]
```diff
@@ -42,7 +42,7 @@
     </xsl:copy>
 </xsl:template>
 
-<xsl:template match="p" xml:space="preserve">
```
Comment (cjp256) [inline snapcraft/extractors/appstream.py]: intentional?

Comment (sergiusens) [reply snapcraft/extractors/appstream.py]: yes, the newer libxml works as intended and this makes all the spaces in this block count

File: spread.yaml (hunk 1 above)
Comment (sergiusens) [reply spread.yaml]: Seem to match indentation of the apt-get line, am I missing anything?

**Pull request merged**